src/screens/Map/MapScreen.tsx

```diff
@@ -34,6 +34,18 @@ export const MapScreen = ({navigation}: MapScreenProps<'MapScreen'>) => {
       activeItemIndex: 0,
     });
   };
+  const navigateToTripSearch: TravelFromAndToLocationsCallback = (
+    location,
+    destination,
+  ) => {
+    navigation.navigate({
+      name: 'TripSearch' as any,
```

Comment:
Shouldn't use `any` here. Needs to go to `TripSearch` through the `Dashboard`. 
```
navigation.navigate('Dashboard', {
  screen: 'TripSearch'
  params: ... 
})
```

Reply:
@gorandalum how do I navigate to `TripSearch` through the `Dashboard` and continue using the merge feature in params as is now?

Reply:
Oh, yeah, it is the other `navigate`-method when using `merge`. You can try something like this then:
```
navigation.navigate({
  name: 'Dashboard',
  params: {
    screen: 'TripSearch',
    params: ...
  },
  merge: true,
})
```